### PR TITLE
New Changelog

### PR DESCRIPTION
**Pull request by Cahllagerfeld**

### CODE REVIEW

**Comment by akosyakov** [comment]
@Cahllagerfeld IDE PR is here: https://github.com/gitpod-io/website/pull/2505

**Comment by Cahllagerfeld** [comment]
Oh I was just about to push the fix for it :D Seems like you've been faster 😄 👍

**Comment by geropl** [comment]
@Cahllagerfeld Here's a PR for WebApp: https://github.com/gitpod-io/website/pull/2528

Will ask the team quickly, but nearly 100% of our work goes into UBP, which is not visible to customers, yet. Not sure if this warrants a change-log entry, yet? Or do we want to tease here, already? :thinking:  @MichaelAring

File: src/lib/contents/changelog/2022-08-01.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+5. Improved operating system support via OpenSSH
+
+### Fixes and improvements
+
+- [#11642](https://github.com/gitpod-io/gitpod/pull/11642) - [local-preview] Add separated anonymous telemetry <Contributors usernames="MrSimonEmms,Pothulapati" />
+- [#11592](https://github.com/gitpod-io/gitpod/pull/11592) - Replce workspace search alert <Contributors usernames="andrew-farries,easyCZ,geropl,gtsiolis" />
+- [#11652](https://github.com/gitpod-io/gitpod/pull/11652) - [dashboard] Fix persistence of checkbox values on settings page <Contributors usernames="andrew-farries,easyCZ" />
+- [#11698](https://github.com/gitpod-io/gitpod/pull/11698) - [kots]: put the "run" collectors into the active namespace <Contributors usernames="MrSimonEmms,adrienthebo" />
+- [#11630](https://github.com/gitpod-io/gitpod/pull/11630) - fix: new project widget broken if 'null' item(s) received from gh api <Contributors usernames="andrew-farries,gtsiolis,laushinka,lucasvaltl,meysholdt,szab100" />
+- [#11508](https://github.com/gitpod-io/gitpod/pull/11508) - Configure team's spending limit on its Billing page. <Contributors usernames="AlexTugarev,easyCZ" />
```
Comment (gtsiolis) [inline src/lib/contents/changelog/2022-08-01.md:49]: **thought**: I'd remove `L49` and `L51` as this is still not visible to all users, but I'm looking forward to the day we no longer spend time curating commits added here.

**Review by gtsiolis — APPROVED** [review]
Looks mergeable! 🏁 

We should probably align with https://github.com/gitpod-io/website/pull/2528 or close one of these two?

Cc @akosyakov @geropl

File: src/lib/contents/changelog/2022-08-01.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: Latest improvements to VS Code
+excerpt: A new ports explorer for VS Code Browser and connection improvements for VS Code Desktop
+date: 2022-08-01
+image: 2022-08-01.jpg
```
[the remark this answers is not in the document]
Comment (akosyakov) [reply src/lib/contents/changelog/2022-08-01.md:5]: How did you optimize it?

File: src/lib/contents/changelog/2022-08-01.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- **[New ports explorer in VS Code Browser](#new-ports-explorer-in-vs-code-browser)** to align interface with VS Code Desktop.
+- **[Connection improvements for VS Code Desktop](#connection-improvements-for-vs-code-desktop)** to bring stability and performance.
+
+> You can enable `Latest Release` in [your preferences](https://gitpod.io/preferences) and try the latest updates as soon as they are available.
+
+### New ports explorer in VS Code Browser
+
+We developed a new ports explorer in VS Code Browser to align the interface with VS Code Desktop.
+This feature can be enabled by setting `gitpod.experimental.portsView.enabled` to `true` in VS Code settings.
+Give it a try and we’d love to hear [your feedback](https://github.com/gitpod-io/gitpod/issues/7408).
```
[the remark this answers is not in the document]
Comment (akosyakov) [reply src/lib/contents/changelog/2022-08-01.md:27]: agree, in this case i think it is alright, feature is not enabled for all users yet, there will be more work

**Comment by geropl** [comment]
> We should probably align with https://github.com/gitpod-io/website/pull/2528 or close one of these two?

I merge my changes, those were only small corrections for WebApp tasks.

File: src/lib/contents/changelog/2022-08-01.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gtsiolis) [reply src/lib/contents/changelog/2022-08-01.md:5]: Using an image compression tool. One of the best, if not the best, out there, is **[squoosh.app](https://squoosh.app/).**

**Pull request merged**